earthaccess/search.py

```diff
@@ -106,6 +115,39 @@ def doi(self, doi: str) -> Type[CollectionQuery]:
         self.params["doi"] = doi
         return self
 
+    def instrument(self, instrument: str) -> Type[CollectionQuery]:
+        """Searh datasets by instrument
+
+        ???+ Tip
+            Not all datasets have an associated instrument. This works
+            only at the dataset level but not the granule (data) level.
+
+        Parameters:
+            instrument (String): instrument of a datasets, e.g. instrument=GEDI
+        """
+        if not isinstance(instrument, str):
+            raise TypeError("instrument must be of type str")
+
+        self.params["instrument"] = instrument
+        return self
+
+    def project(self, project: str) -> Type[CollectionQuery]:
+        """Searh datasets by associated project
+
+        ???+ Tip
+            Not all datasets have an associated project. This works
+            only at the dataset level but not the granule (data) level.
+            Will bring datasets across DAACs matching the project.
```

Comment:
```suggestion
            Will return datasets across DAACs matching the project.
```
Bring reads weird here to me